src/Status/SettingsParser.tsx

```
@@ -0,0 +1,43 @@
+        case 'paragraph':
+            return <Paragraph key={ element.hook_key } element={ element } />;
+        case 'link':
+            return <Link key={ element.hook_key } element={ element } />;
+        case 'button':
+            return <Button key={ element.hook_key } element={ element } />;
+
+        default:
+            // @ts-ignore
+            return wp.applyFilters(
+                'dokan_status_custom_element',
+                <></>,
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.

(lint/complexity/noUselessFragments)